### PR TITLE
Arm

### PR DESCRIPTION
**Pull request by yvesfasel**
modifed following files for arm compability:

- etl/include/etl/vectorization.hpp : line 17
- etl/include/etl/impl/vec/gemm_blis.hpp: line 255
- include/dll/layer.hpp: line 51

### CODE REVIEW

**Comment by yvesfasel** [comment]
conditional compiling for arm architecture

File: .gitmodules [hunk 1]
```diff
@@ -20,3 +16,7 @@
 [submodule "cifar-10"]
 	path = cifar-10
 	url = https://github.com/wichtounet/cifar-10.git
+[submodule "etl"]
```
Comment (wichtounet) [inline .gitmodules:19]: The change to the submodule should not be part of this PR.

File: include/dll/layer.hpp [hunk 2]
```diff
@@ -48,10 +48,15 @@ struct layer {
      * \brief Default initialize the layer
      */
     layer() {
+#ifdef __ARM_ARCH
+ // do not set flush zero mode
+#else
```
Comment (wichtounet) [inline include/dll/layer.hpp:53]: Instead of having an empty branch, you can make it simpler by using #ifndef

**Comment by wichtounet** [comment]
Also, your description include three files but the PR only include one of them. Is that normal?

**Comment by yvesfasel** [comment]
> 
> 
> Also, your description include three files but the PR only include one of them. Is that normal?

no, it should include those 3 files:

    etl/include/etl/vectorization.hpp 
    etl/include/etl/impl/vec/gemm_blis.hpp
    include/dll/layer.hpp

File: .gitmodules [hunk 3]
```diff
@@ -2,10 +2,6 @@
 	path = make-utils
 	url = https://github.com/wichtounet/make-utils.git
 	branch = master
-[submodule "etl"]
-	path = etl
-	url = https://github.com/wichtounet/etl.git
-	branch = master
```
Comment (wichtounet) [inline .gitmodules:8]: This change should not be part of the PR and the etl update as well.
